eth/tracers/live/live_api_eth.go

```diff
@@ -0,0 +1,40 @@
+package live
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/tracing"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
+)
+
+type ethAPI struct {
+	backend tracing.Backend
+	live    *live
+}
+
+func (n *ethAPI) GetTransactionBySenderAndNonce(ctx context.Context, sender common.Address, nonce hexutil.Uint) (*ethapi.RPCTransaction, error) {
+	// TODO:
+	// 1. return nil if sender is a contract
+	// 2. check with txpool first
+	txHash, err := n.live.kvdb.Get(append(sender.Bytes(), encodeNumber(uint64(nonce))...))
+	if err != nil {
+		return nil, nil
+	}
+
+	found, tx, blockHash, blockNumber, index, err := n.backend.GetTransaction(ctx, common.BytesToHash(txHash))
```

Comment:
Myeah I thought we would just return the txhash? Is any other client returning full tx objects?

Reply:
I'm following the RPC in https://github.com/manifoldfinance/rpc-eip-drafts/blob/master/eth_getTransactionBySenderAndNonce/README.md, so in reth, I return the full tx, ref https://github.com/paradigmxyz/reth/pull/10540.

I think I might need to reconsider whether to return the txhash or full tx, I'll take a look at the behavior of other clients or providers again. Indeed returning the txhash is more simple and then sdk's side can fetching the tx body or receipt by calling the `eth_getTransaction` or `eth_getTransactionReceipt`.